### PR TITLE
Add fresh option to dockerdev

### PR DESCRIPTION
**Pull request by joejulian**
Add an option to freshen the docker image when running dockerdev.

### CODE REVIEW

**Comment by joejulian** [comment]
requires changes in PR #596

File: hack/dockerdev [hunk 1]
```diff
@@ -14,6 +14,19 @@ realpath() {
...
+    echo ""
+    echo " Options:"
+    echo "     -c,--config <filename>    config.yaml file to use"
+    echo "                               default: $HOME/.kraken/default.yaml"
+    echo "     -h,--help                 this text"
+    echo "     -i,--image   <imagename>  k2-tools docker image to use"
+    echo "                               default: quay.io/samsung_cnct/k2-tools"
+    echo "     -m,--home <path>          specify the path to be used for \$HOME"
+}
+
```
Comment (mikeln) [inline hack/dockerdev]: Missing freshen

File: hack/dockerdev [hunk 2]
```diff
@@ -71,6 +92,11 @@ MAPPINGS="${MAPPINGS} -v ${MYHOME}/.aws:${MYHOME}/.aws"
 # Map a bashrc file
 MAPPINGS="${MAPPINGS} -v ${GITROOT}/lib/bashrc:${MYHOME}/.bashrc"
 
+if [ -n "${FRESHEN}" ]; then
+    echo "Refreshing image: ${IMAGE}"
+    docker pull ${IMAGE}
```
Comment (mikeln) [inline hack/dockerdev]: Will this always pull the image, if the version isn't `latest`?
We might need to `docker rmi` the image first.

Comment (mikeln) [inline hack/dockerdev]: probably ok, looks like Docker got their act together on this and check the layer hashes for diffs.

Comment (joejulian) [inline hack/dockerdev]: According to the docker documentation:

> When using tags, you can docker pull an image again to make sure you have the most up-to-date version of that image. For example, docker pull ubuntu:14.04 pulls the latest version of the Ubuntu 14.04 image.

https://docs.docker.com/engine/reference/commandline/pull/#pull-an-image-by-digest-immutable-identifier

**Pull request merged**